jupyterlab_caip_optimizer/src/setupTests.ts

```diff
@@ -38,6 +38,21 @@ beforeAll(() => {
     }
     return originalError(...args);
   });
+
+  // https://github.com/mui-org/material-ui/issues/15726#issuecomment-493124813
+  // TODO: remove upgrading to jest v26
+  global.document.createRange = () => ({
```

Comment:
This resolved some testing issues that came up due to animations with material ui and the autocomplete component.